docs/examples/example.json

```diff
@@ -191,7 +191,29 @@
             "url": "https://www.example.com"
           },
           "roles": [
-            "supplier"
+            "supplier",
```

Comment:
Remove the 'supplier' role as this party is not listed in `contractingProcesses.summary.suppliers`

Reply:
The party with id `GB-COH-11111111` is listed in `contractingProcess.summary.suppliers` for ocid = ocds-a1b1c1-a410a80d-adc8-11e6-9901-0019b9f3037b.

Reply:
Sorry, I commented this on the wrong row. I'll add the comment in the correct place